ftdetect/julia.vim

```diff
@@ -0,0 +1 @@
+autocmd BufRead,BufNewFile *.jl setfiletype=julia
```

Comment:
hmm I think it needs to be `setfiletype julia`

Reply:
Right, sorry, this is why I should do this kind of things during the week. BTW, the other files use `setlocal filetype` instead. Maybe that's a better option? My understanding is `setfiletype` only works if the file type has not been set already, which seems nicer, but I'm not sure, maybe it doesn't matter